lib/phlex/rails/renderable.rb

```diff
@@ -37,6 +39,38 @@ def render_in(view_context, &block)
 					call(view_context: view_context).html_safe
 				end
 			end
+
+			def <<(value)
+				@_target << value
+			end
+
+			def safe_append=(value)
+				return unless value
+
+				self << case value
```

Comment:
We can append to `@_target` directly here, saving a method call.

```suggestion
				@_target << case value
```